Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```diff
@@ -32,3 +32,16 @@
     contents:
     - id: BoxSurvivalMedical
     - id: RubberStampPsychologist
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackCaptain
+  id: ClothingBackpackAdminAssistantFilled
+  name: administrative assistant's backpack
+  description: Its a backpack build for carrying paper.
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
+      - id: Flash
```

Comment:
Flash? They're an assistant?

Reply:
Was thinking they'd need some self defence on the bridge, but I guess they can just take the funny bridge axe in bad situations. Flash removedm